param/parameterized.py

```diff
@@ -1137,6 +1138,48 @@ def watch(self_,parameter_name,parameter_attribute=None,fn=None):
             subscribers[parameter_attribute].append(fn)
 
 
+    # TODO:
+    #
+    #   * error if not watched? (would then need a way to ask about status)
+    #
+    #   * would it be better to have watch() return an ID which could
+    #     later be used to remove?
+    #
+    #   * this method deliberately doesn't provide an easy way to
+    #     unwatch things declared at class level (i.e. can't turn off
+    #     watching for @depends(...,watch=True)). Does that make
+    #     sense?
```

Comment:
That seems very reasonable, imo part of the class definition and should not be messed with.